src/panels/config/automation/ha-automation-trace.ts

```diff
@@ -350,7 +350,11 @@ export class HaAutomationTrace extends LitElement {
   }
 
   private async _loadTraces(runId?: string) {
-    this._traces = await loadTraces(this.hass, "automation", this.automationId);
+    this._traces = await loadTraces(
+      this.hass,
+      "automation",
+      decodeURIComponent(this.automationId)
```

Comment:
How come that it is encoded here?

Reply:
It's because of this:
https://github.com/home-assistant/frontend/blob/dev/src/panels/config/automation/ha-config-automation.ts#L91

Should we decode there instead?